src/views/groups/GroupEdit/components/EditGroupCredentials.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div>
+    <p style="font-weight:bold">{{ $t('edit-group:privacy') }}</p>
+      <b-form-checkbox style="margin-top:15px" :checked="publicCertificate" @change="toggleCertificate">
+      <b>{{ $t('edit-group:private') }}</b>
+    </b-form-checkbox>
+  </div>
+</template>
+
+<script lang="ts">
+  import { Component, Vue, Prop } from 'vue-property-decorator';
+  import EditField from '@/components/Common/EditField.vue';
+
+  @Component({
+    components: { EditField },
+  })
+  export default class GroupEditCredentials extends Vue {
```

Comment:
Can we come up with a better name than "credentials"? Doesn't seem properly descriptive